package.json

```diff
@@ -66,21 +66,18 @@
     "@hapi/hapi": "^18.3.2",
     "@hapi/joi": "^15.1.1",
     "debug": "^4.1.1",
-    "dexie": "^2.0.4",
     "execa": "^2.0.4",
     "fs-extra": "^8.1.0",
     "hat": "~0.0.3",
-    "ipfs-http-client": "^36.0.0",
+    "ipfs-http-client": "^36.1.0",
```

Comment:
`v37` is out - breaking change is minor, just downgrades `multiaddr` from `v7` to `v6` until we can upgrade all `js-ipfs*` in one go.